### PR TITLE
NDI-5: Styled Manage event page

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
### Description:
This PR styles Event Dashboard page

#### Before:
before

#### After:
after

### Technical Details
We are fixing action-link buttons height in this PR to adapt normal height instead of fixed 27px.
https://github.com/civicrm/org.civicrm.shoreditch/pull/96/files#diff-dcce3294b88d7236dde50eace88b572bL296
```
.action-link {
 -    line-height: 45px;
 +    height: auto;
```
### Notes:
- [x] BackstopJS tested
- [x] BrowserStack tested

#### For Tester:
- This PR also covers NDI-6
- This PR depends on https://github.com/civicrm/org.civicrm.shoreditch/pull/94 for new color palette.

### CODE REVIEW

**Comment by AkA84** [comment]
@mukeshcompucorp 

> This PR depends on #94 for action link height (Add Event button) 

why? #94 is supposed to be just about colors

File: scss/civicrm/contact/pages/_events.scss [hunk 1]
```diff
@@ -680,3 +680,84 @@
...
+    h3 {
+      @include panel-header();
+    }
+
+    .form-layout{
+      margin: 20px;
+    }
+  }
+
+  div#alpha-filter {
```
[the remark this answers is not in the document]
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_events.scss]: Yes, this is to overcome specificity on civicrm default style. or we need to use !important.

File: scss/civicrm/common/_titles.scss [hunk 2]
```diff
@@ -7,3 +7,12 @@
   box-sizing: border-box;
   background: transparent;
 }
+
+@mixin panel-header() {
```
[the remark this answers is not in the document]
Comment (mukeshcompucorp) [reply scss/civicrm/common/_titles.scss]: yes, this will be used again on NDI-4 as well and other dashboard pages.

**Comment by mukeshcompucorp** [comment]
> why? #94 is supposed to be just about colors
Yes, but, this was just very minor point to make height: auto for any buttons breaking design. (1 line change) to make many section proper to look

File: scss/bootstrap/overrides/style/_type.scss [hunk 3]
```diff
@@ -3,7 +3,10 @@
 
 h1, .h1 { font-weight: $crm-font-weight-h1; }
 h2, .h2 { font-weight: $crm-font-weight-h2; }
-h3, .h3 { font-weight: $crm-font-weight-h3; }
+h3, .h3 {
```
[the remark this answers is not in the document]
Comment (mukeshcompucorp) [reply scss/bootstrap/overrides/style/_type.scss]: On every place we should have font-weight bold for h3 tag. that's why I changed here.

**Pull request merged**